packages/domain-picker/tsconfig.json

```diff
@@ -0,0 +1,35 @@
+{
+	"compilerOptions": {
+		"target": "ES5",
+		"lib": [ "DOM", "ES2015" ],
+
+		"baseUrl": ".",
+		"module": "esnext",
+		"allowJs": false,
+		"jsx": "react",
+		"declaration": true,
+		"declarationDir": "dist/types",
+		"outDir": "dist/esm",
+		"rootDir": "src",
+
+		"strict": true,
+		"noUnusedLocals": true,
+		"noUnusedParameters": true,
+		"noImplicitReturns": true,
+		"noFallthroughCasesInSwitch": true,
+
+		"moduleResolution": "node",
+		"esModuleInterop": true,
+
+		"forceConsistentCasingInFileNames": true,
+
+		"typeRoots": [ "../../node_modules/@types" ],
+		"types": [],
+
+		"importHelpers": true,
+		"composite": true,
+		"tsBuildInfoFile": "../../.tsc-cache/packages__domain-picker"
+	},
+	"include": [ "src" ],
+	"exclude": [ "**/docs/*", "**/test/*" ]
```

Comment:
Since we don't have `docs` maybe that can be removed here?

Reply:
Removed.